### ISSUE TITLE
Unable access new plugins and themes from wordpress

### ISSUE THREAD

**Issue by yashashebbarbail** [lando/lando#1678]
Wordpress is running fine. But it is not accessing the internet to load new plugins and themes. I am getting error when I access this page http://localhost:32789/wordpress/wp-admin/plugin-install.php
![localhost_32789_wordpress_wp-admin_plugin-install php(Laptop with MDPI screen)](https://user-images.githubusercontent.com/29854066/59654946-0f671500-91b6-11e9-81c0-22c388b67726.png)

> An unexpected error occurred. Something may be wrong with WordPress.org or this server’s configuration. If you continue to have problems, please try the support forums.

.lando.yml :
```
name: web-server
recipe: lamp
config:
  webroot: www
  database: mysql:5.7
  php: '7.3'
services:
  phpmyadmin:
    type: phpmyadmin:4.7
    hosts:
      - database
```

**Comment by yashas-bhandary** [comment]
Found a alternative way to resolve this issue.
Wordpress support - [Error: WordPress could not establish a secure connection to WordPress.org](https://wordpress.org/support/topic/error-wordpress-could-not-establish-a-secure-connection-to-wordpress-org/)
Installing [SaFly Curl Patch](https://wordpress.org/plugins/safly-curl-patch/) solved the problem.

Please let me know if there is alternative solution.

**Comment by yashashebbarbail** [comment]
https://github.com/lando/lando/issues/1695

**Comment by leoloso** [comment]
@yashashebbarabailu Have you been able to fix it? I'm having the same issue: #2913

**Comment by yashashebbarabailu** [comment]
> @yashashebbarabailu Have you been able to fix it? I'm having the same issue: #2913

I didn't find any solution. I am not using lando anymore.
